test/format/format_config.c

```diff
@@ -1266,6 +1393,12 @@ config_statistics(void)
 static void
 config_transaction(void)
 {
+    /* Predictable replay requires timestamps. */
+    if (GV(RUNS_PREDICTABLE_REPLAY)) {
+        config_single(NULL, "transaction.timestamps=on", true);
+        config_single(NULL, "transaction.implicit=0", false);
```

Comment:
Alphabetize these two lines.

Reply:
This is still not alphabetized. Is there a reason requiring this order?

Reply:
Just fixed - sorry I did not see your earlier comment.